### PR TITLE
I update CONTRIBUTING docs.

### PR DESCRIPTION
**Pull request by SweetmanTech**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
...
+    of permissions in the URL reads 536637602902.
 11. Send the URL via Discord DM to nonsense to have it added to the test
     Discord server
 
 ## Clone the repo to local
 
-1. Create a new feature branch from the dev branch
-2. Copy the .env file and name it whatever you want; add it to your local
-   .gitignore
+1. Create a new feature branch from the dev branch (`docs/<name>`, `feature/<name>`, `release/<name>`, `patch/<name>`).
```
Comment (amaredeus) [inline docs/CONTRIBUTING.md:36]: We opted to go with `hotfix/<name>` instead of `patch/<name>`

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
...
+2. Copy the `.env.template` file and name it `.env`;
 3. Request the test Notion token and MongoDB URI from one of the repo
    maintainers, copy your bot's token (found under the bot tab in the
    developer's portal) and replace the relevant values in your copied .env
    file, making sure to update the file name where it is required in `app.js`
-4. `npm install`
-5. Run your bot and test your connections
+4. `yarn`
```
Comment (amaredeus) [inline docs/CONTRIBUTING.md:42]: `yarn install` or even simpler doing `yarn start` will also run the prestart script to get everything installed and running

**Review by amaredeus — CHANGES_REQUESTED** [review]
Awesome work on the jobs, just small minor suggestions

**Comment by amaredeus** [comment]
@SweetmanTech Looks like build is failing because branch SweetmanTech:docs/onboarding is not in sync with Bankless:develop branch. Can you try running 

`git pull origin develop` where origin is git url from Bankless?

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (SweetmanTech) [reply docs/CONTRIBUTING.md:36]: updated ✅

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (SweetmanTech) [reply docs/CONTRIBUTING.md]: updated ✅

**Comment by SweetmanTech** [comment]
Addressed comments ✅ 

sorry for the delay @nonsensecodes & @amaredeus I just finished moving back to the US from Thailand and as soon as I got back I got COVID but I'm recovering now and getting back up to speed with everything!

Let me know if there's any other feedback I can address here. 🙏

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
...
 -   Manage Channels
 -   Kick Members
 -   Ban Members
--   Manage Emojis
+-   Manage Emojis & Stickers
 -   Manage Webhooks
 -   View Channels
 -   All of the permissions under "Text Permissions"
--   No permissions under "Voice Permissions"
+-   Only permissions under "Voice Permissions" are Connect & Use Voice Activity.
```
Comment (amaredeus) [inline docs/CONTRIBUTING.md:27]: Voice permissions are not  needed. We can revert this change

Comment (amaredeus) [inline docs/CONTRIBUTING.md:27]: Voice permissions are not needed. We can revert this change

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -16,33 +16,32 @@ To get setup, please follow these guidelines:
...
 
 10. Select and copy the generated URL under "Scopes." Make sure the value
-    of permissions in the URL reads 4027055222
+    of permissions in the URL reads 536637602902.
```
Comment (amaredeus) [inline docs/CONTRIBUTING.md:30]: since this value can change based on needs, can we have a link here to the PERMISSIONS file in docs?

**Review by amaredeus — CHANGES_REQUESTED** [review]

File: docs/CONTRIBUTING.md (hunk 3 above)
Comment (SweetmanTech) [reply docs/CONTRIBUTING.md:27]: rolled back changes here ✅

File: docs/CONTRIBUTING.md (hunk 4 above)
Comment (SweetmanTech) [reply docs/CONTRIBUTING.md:30]: great idea 💡

Comment (SweetmanTech) [reply docs/CONTRIBUTING.md:30]: updated ✅

**Review by amaredeus — APPROVED** [review]

**Review by nonsensecodes — APPROVED** [review]
As it appears the requested changes were implemented, I'm approving this PR. Thank you @SweetmanTech for following through and making the edits, and thank you @amaredeus for reviewing and suggesting said changes. Great job!
